Source/Tools/LinuxEmulation/LinuxSyscalls/GdbServer.h

```diff
@@ -35,8 +35,10 @@ class GdbServer {
     LibraryMapChanged = true;
   }
 
+  void CreateThreadCallback(FEX::HLE::ThreadStateObject* ThreadObject);
```

Comment:
```suggestion
  void OnThreadCreated(FEX::HLE::ThreadStateObject* ThreadObject);
```

Avoids confusion as to what is being created.